scout/server/blueprints/variants/templates/variants/sv-variants.html

```diff
@@ -222,6 +222,11 @@
       return true;
     }
 
+    function promptFilterName(suggestion) {
```

Comment:
Here also, it would be better to do it with a modal. A modal placed in utils.html which will be shared by all 3 types of variants perhaps?

Reply:
Sure, that has merit. It is a bit more code in total, but elegant in terms of final simplicity.